LogosLinuxInstaller.py

```diff
@@ -293,17 +305,20 @@ def main():
 
     ### Run desired action (requested function, defaulting to installer)
     # Run safety checks.
-    utils.die_if_running()
+    # FIXME: Fix utils.die_if_running() for GUI; as it is, it breaks GUI self-update when updating LLI as it asks
+    # for a confirmation in the CLI. Disabled until it can be fixed. Avoid running multiple instances of the program.
+    # utils.die_if_running()
     utils.die_if_root()
 
     # Print terminal banner
-    logging.info(f"{config.LOGOS_SCRIPT_TITLE}, {config.LOGOS_SCRIPT_VERSION} by {config.LOGOS_SCRIPT_AUTHOR}.")
+    logging.info(f"{config.LLI_TITLE}, {config.LLI_CURRENT_VERSION} by {config.LLI_AUTHOR}.")
     logging.debug(f"Installer log file: {config.LOGOS_LOG}")
 
-    utils.self_update()
+    utils.check_for_updates()
 
     # Check if app is installed.
-    install_not_required = [ 'remove_install_dir', 'check_dependencies', 'update_to_latest_recommended_appimage', 'set_appimage_symlink' ]
+    install_not_required = ['remove_install_dir', 'check_dependencies',  'update_latest_lli_release',
+                            'update_to_latest_recommended_appimage', 'set_appimage_symlink']
```

Comment:
`update_latest_lli_release` needs to be ` update_to_latest_lli_release` to match the function name